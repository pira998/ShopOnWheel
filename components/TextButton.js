import React from "react";

import {Animated,ScrollView,StyleSheet,TouchableWithoutFeedback,Modal ,View,TouchableOpacity,Image,Text } from "react-native";

import { COLORS,FONTS,SIZES,icons,constants } from "../constants";

const TextButton =({label,labelStyle,buttonContainerStyle,onPress})=>{
return(
    <TouchableOpacity
    style={{
        alignItems:'center',
        justifyContent:'center',
        backgroundColor:COLORS.primary,
        ...buttonContainerStyle
    }}
    onPress={onPress}
    >
        <Text
        style={{
            color:COLORS.white,...FONTS.h3,...labelStyle
        }}>
            {label}
        </Text>

    </TouchableOpacity>
)
}

export default TextButton