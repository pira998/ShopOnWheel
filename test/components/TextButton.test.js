import React from 'react';
import renderer from 'react-test-renderer';
import TextButton from '../../components/AuthTextButton';


test('renders correctly', () => {
  const tree = renderer.create(<TextButton />).toJSON();
  expect(tree).toMatchSnapshot();
});