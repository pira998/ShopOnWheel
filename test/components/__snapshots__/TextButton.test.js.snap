// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FF6C44",
      "justifyContent": "center",
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "sans-serif",
        "fontSize": 16,
        "lineHeight": 22,
      }
    }
  />
</View>
`;
